spec/order_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'order'
+
+describe Order do
+  subject(:order) { described_class.new }
+  let(:bill) { 20 }
+  let(:twilio) { double :twilio}
+  let(:takeaway) { double :takeaway}
+  let(:items) { { pies: 5, chips: 3} }
+  let(:confirm) {"Thank you! Your order was placed "\
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.